cmd/tether/attach.go

```diff
@@ -328,3 +364,11 @@ func (t *attachServerSSH) channelMux(in <-chan *ssh.Request, process *os.Process
 
 	detach()
 }
+
```

Comment:
vendor/github.com/docker/docker/pkg/term/term.go seems have the definitions as well as resize functionality